openapi_python_client/parser/properties/__init__.py

```diff
@@ -544,6 +554,16 @@ def build_schemas(*, components: Dict[str, Union[oai.Reference, oai.Schema]]) ->
                 schemas = schemas_or_err
                 processing = True  # We made some progress this round, do another after it's done
         to_process = next_round
-    schemas.errors.extend(errors)
 
+    resolve_errors: List[PropertyError] = []
+    models = list(schemas.models.values())
+    for model in models:
```

Comment:
```suggestion
    for model in schemas.models.values():
```

I don't see a reason to construct a new list just for iterating.